docs/readthedocs/source/doc/Orca/QuickStart/orca-keras-quickstart.md

```diff
@@ -38,7 +38,13 @@ elif cluster_mode == "yarn":  # For Hadoop/YARN cluster
 
 This is the only place where you need to specify local or distributed mode. View [Orca Context](./../Overview/orca-context.md) for more details.
 
-**Note:** You should `export HADOOP_CONF_DIR=/path/to/hadoop/conf/dir` when running on Hadoop YARN cluster. View [Hadoop User Guide](./../../UserGuide/hadoop.md) for more details.
+**Note:** You should correctly set HADOOP_HOME, HADOOP_CONF_DIR, HADOOP_HDFS_HOME, LD_LIBRARY_PATH, etc when running on Hadoop YARN cluster.
```

Comment:
How about, "To use tensorflow_datasets on HDFS, you should correctly set HADOOP_HOME, HADOOP_CONF_DIR, HADOOP_HDFS_HOME, LD_LIBRARY_PATH, etc. For more details, please refer to TensorFlow documentation [link](https://github.com/tensorflow/docs/blob/r1.11/site/en/deploy/hadoop.md),"

Reply:
updated.